app/javascript/packs/application.js

```diff
@@ -9,7 +9,9 @@
 
 import "jquery"
 import "jquery-ujs"
+import PolynomialRegression from "../../../node_modules/js-polynomial-regression/dist/PolynomialRegression"
```

Comment:
Instead of importing this as global, it should be instead imported into the files that need it.  The reason I imported all these to global was so that non-webpack js that was still served through the asset pipeline would continue working while the js was slowly transitioned.  It seemed to be a common approach to moving off sprockets for the js.

Reply:
Actually I didn't end up using this dependency, so I'll remove it. But when I was trying to use it, I was running into an issue I think due to the location of the dist. webpacker was giving me errors that it couldn't be found when importing until I did this. I think I just don't understand webpack or webpacker at all.